src/GlobalStyles.js

```diff
@@ -1,13 +1,26 @@
 import { createGlobalStyle } from 'styled-components'
 
 const GlobalStyles = createGlobalStyle`
```

Comment:
Prettier doesn't work for `createGlobalStyle`. You can replace it with `css`, save it (it's formatted now) and replace it again with `createGlobalStyle`.